Policies/github-terms-of-service.md

```diff
@@ -133,20 +134,22 @@ You may create or upload User-Generated Content while using the Service. You are
 #### 2. GitHub May Remove Content
 We do not pre-screen User-Generated Content, but we have the right (though not the obligation) to refuse or remove any User-Generated Content that, in our sole discretion, violates any GitHub terms or policies.
 
-#### 3. Ownership of Content, Right to Post, and License Grants
-You retain ownership of and responsibility for Content you create or own ("Your Content"). If you're posting anything you did not create yourself or do not own the rights to, you agree that you are responsible for any Content you post; that you will only submit Content that you have the right to post; and that you will fully comply with any third party licenses relating to Content you post.
+#### 3. Ownership of Content and Right to Post
+Posting Content does not effect its ownership. You may only submit Content that you have the right to post, and You must fully comply with any third party licenses relating to Content you post.
 
-Because you retain ownership of and responsibility for Your Content, we need you to grant us — and other GitHub Users — certain legal permissions, listed in Sections D.4 — D.7. These license grants apply to Your Content. If you upload Content that already comes with a license granting GitHub the permissions we need to run our Service, no additional license is required. You understand that you will not receive any payment for any of the rights granted in Sections D.4 — D.7. The licenses you grant to us will end when you remove Your Content from our servers, unless other Users have forked it.
+Because GitHub does not User-Generated Content, We — and Other Users — need certain legal permissions, as listed in Sections D.4 — D.7. You understand that You will not receive any payment for any of the rights discussed in Sections D.4 — D.7. Any permissions You grant to Us and Other Users will end when You remove Your Content from our servers.
```

Comment:
> The first new sentence is lacking a verb.

The verb is “need”.

> perhaps “Any permissions You grant to Us and Other Users under §D.4–D.7”?

That's better.  I'll add it to v2.  I'll also change the em-dashes to [en-dashes for the range][1].

[1]: https://en.wikipedia.org/wiki/Dash#Ranges_of_values